Jenkinsfile

```diff
@@ -124,7 +124,7 @@ def buildInstaller() {
     sh "yarn --frozen-lockfile --force"
     sh "rm -rf ./${distFolder}"
     sshagent(['projects-storage.eclipse.org-bot-ssh']) {
-        sh "yarn package"
+        sh "yarn deploy"
```

Comment:
Why is this changed? Won't this try to publish the build package to the generic server during the build?

Reply:
The generic build server does not have an automated publish feature like github. 
So this will only produce the required metadata and bundle it accordingly.